packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -65,6 +68,7 @@ export const Alert: React.FunctionComponent<AlertProps> = ({
   onTimeout,
   truncateTitle = 0,
   tooltipPosition,
+  customIcon = undefined,
```

Comment:
```suggestion
  customIcon,
```
it's undefined by default